.travis.yml

```diff
@@ -95,9 +95,9 @@ matrix:
 
         # Now try with all optional dependencies.
         - language: python
-          python: 3.8
+          python: nightly
           dist: bionic
-          name: Python 3.8 with all optional dependencies
+          name: Python dev with all optional dependencies
```

Comment:
I would go with mandatory dependencies only. We should be able to test even when some of our upstream have issues

Reply:
also, you need to update the tox stuff below